packages/react-devtools-shared/src/devtools/views/ErrorBoundary/ReportNewIssue.js

```diff
@@ -13,9 +13,18 @@ import {searchGitHubIssuesURL} from './githubAPI';
 import styles from './shared.css';
 
 function encodeURIWrapper(string: string): string {
-  return encodeURI(string).replace(/#/g, '%23');
```

Comment:
Any reason you didn't use `encodeURIComponent(string)` instead which has `replace(/#/g, '%23')` built in?

Reply:
I _think_ I used `encodeURIComponent` initially and it _didn't work_ for some reason so I backed it out 😅  but maybe I should try again to verify.